include/mesos/mesos.proto

```diff
@@ -1888,6 +1888,33 @@ message ResourceStatistics {
   optional uint64 net_tx_burst_rate_limit = 47;
   optional uint64 net_tx_burst_size = 48;
 
+  message RatePercentiles {
+    optional uint64 min = 1;
+    optional uint64 max = 2;
+    optional uint64 p50 = 3;
+    optional uint64 p90 = 4;
+    optional uint64 p95 = 5;
+    optional uint64 p99 = 6;
+    optional uint64 p999 = 7;
+    optional uint64 p9999 = 8;
+    optional uint64 samples = 9;
+  }
+
+  message RateStatistics {
+    optional RatePercentiles rx_rate = 1;
+    optional RatePercentiles rx_packet_rate = 2;
+    optional RatePercentiles rx_drop_rate = 3;
+    optional RatePercentiles rx_error_rate = 4;
+    optional RatePercentiles tx_rate = 5;
+    optional RatePercentiles tx_packet_rate = 6;
+    optional RatePercentiles tx_drop_rate = 7;
+    optional RatePercentiles tx_error_rate = 8;
+    optional double sampling_window_secs = 9;
+    optional double sampling_interval_secs = 10;
+  }
+
+  optional RateStatistics net_rate_statistics = 51;
+
   // The kernel keeps track of RTT (round-trip time) for its TCP
```

Comment:
Wondering whether these new protobuf messages should be copied into `v1/mesos.proto` to maintain compatibility.

Reply:
Copied over.